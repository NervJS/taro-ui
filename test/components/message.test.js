import Nerv from 'nervjs'
import { renderToString } from 'nerv-server'
import AtMessage from '../../.temp/components/message/index'

describe('AtMessage Snap', () => {
  it('render initial AtMessage', () => {
    const component = renderToString(<AtMessage />)
    expect(component).toMatchSnapshot()
  })

  it('render AtMessage -- props className', () => {
    const component = renderToString(<AtMessage className='test' />)
    expect(component).toMatchSnapshot()
  })

  it('render AtMessage -- props customStyle', () => {
    const component = renderToString(<AtMessage customStyle='color:red;' />)
    expect(component).toMatchSnapshot()
  })
})
