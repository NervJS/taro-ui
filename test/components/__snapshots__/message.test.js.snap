// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AtMessage Snap render AtMessage -- props className 1`] = `"<div class=\\"at-message at-message--hidden at-message--info test\\"></div>"`;

exports[`AtMessage Snap render AtMessage -- props customStyle 1`] = `"<div style=\\"color:red;\\" class=\\"at-message at-message--hidden at-message--info\\"></div>"`;

exports[`AtMessage Snap render initial AtMessage 1`] = `"<div class=\\"at-message at-message--hidden at-message--info\\"></div>"`;
